test/compilable/ctests2.c

```diff
@@ -0,0 +1,122 @@
+/*************************************************/
+// https://issues.dlang.org/show_bug.cgi?id=22304
+
+int * __attribute__((__always_inline__)) foo(void)
+{
+    return 0;
+}
+
+/*************************************************/
+// https://issues.dlang.org/show_bug.cgi?id=22312
```

Comment:
This test looks new

Reply:
I had forgotten to included it in an earlier PR.